src/vuex/getters.js

```diff
@@ -97,6 +97,7 @@ export const bondDenom = getters =>
   (getters.stakingParameters.parameters &&
     getters.stakingParameters.parameters.bond_denom) ||
   `uatom`
+export const minting = state => state.minting
```

Comment:
I don't think this is necessary. I know it's a habit we have, but we can use `mapState` to get at this, without creating another getter.